lxd/daemon.go

```diff
@@ -116,6 +117,12 @@ type Daemon struct {
 
 	ctx    context.Context
 	cancel context.CancelFunc
+
+	// Stores the time the metrics were last fetched. This will be used to prevent stressing the instances too much.
+	metricsLastFetchedTime time.Time
```

Comment:
In my example the other day I changed this to `metricsLastBuildTime` as I feel that better represents this value (as metrics can be requested and served from the cache).

Reply:
Oh, I didn't catch that, sorry.